### DISCUSSION TITLE
Migrating Issues to Discussions - any way to search for Issues that do not reference commits or PRs?

### DISCUSSION THREAD

**Discussion by taligentx** [community/community#39] [Discussions]
Hi folks, thanks for implementing Discussions!  It's perfect for my projects and looking forward to seeing the Issues trackers become dedicated to actual issues instead of how-to questions, general support, etc.

I'm currently migrating the latter types of Issues to Discussions - is there any way to search for Issues that do not contain references by a commit or PR?  The typical how-to questions or other support questions as Issues usually don't have associated commits or PRs so these are the most likely candidates to be migrated as a discussion.  It'd be handy to have a search filter that can identify these types of Issues - thanks!

-Nikhil

**Comment by becca** [comment]
🤗 thank you! 

> is there any way to search for Issues that do not contain references by a commit or PR? 

I believe this is possible! If you take a look at [these docs](https://docs.github.com/en/free-pro-team@latest/github/searching-for-information-on-github/searching-issues-and-pull-requests#search-for-linked-issues-and-pull-requests) there's a way for you to narrow the results down into what you might be looking for. Please let me know if this is sufficient and helpful for you to convert those issues over into Discussions
